.github/workflows/deploy-gcp-tests.yml

```diff
@@ -248,8 +284,27 @@ jobs:
           && \
           docker run ${{ inputs.test_variables }} -t --name ${{ inputs.test_id }} \
           --mount type=volume,src=${{ inputs.disk_prefix }}-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},dst=${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }} \
+          --mount type=volume,src=${{ inputs.disk_prefix }}-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},dst=${{ inputs.root_state_path }}/${{ inputs.lwd_state_dir }} \
```

Comment:
Can we mount the same disk to two different paths?
What if both states use the same file or directory name?

Reply:
> Can we mount the same disk to two different paths?

Yes, this is basically to have both path available in the machine, and allow the test to save to the lightwalletd cached state directory. A harder way is to make two separate disks, but so far we don't need that complexity (which would also duplicate most of the steps)



> What if both states use the same file or directory name?

I don't know if there's a use case which might cause a collision, as tested so far...we don't have those. If this could happen with different tests in the future, we could worry about that in that moment.

Reply:
Just to clear this out, each state it's in it's own directory, as defined by:
`ZEBRA_CACHED_STATE_DIR` and `LIGHTWALLETD_DATA_DIR`, they just share the same root path `/var/cache`

Reply:
Also, this is working with the `lightwalletd_update_sync` which uses both cached states (from different directories)

Reply:
> Just to clear this out, each state is in it's own directory, as defined by:
> `ZEBRA_CACHED_STATE_DIR` and `LIGHTWALLETD_DATA_DIR`, they just share the same root path `/var/cache`

That's not the whole picture.

Inside the docker container, the root directory of the same disk is mapped to two different paths:
* ZEBRA_CACHED_STATE_DIR: /var/cache/zebrad-cache
* LIGHTWALLETD_DATA_DIR: /var/cache/lwd-cache

So in the container filesystem, the paths are separate.

But on the cached disk, all the state data goes into:
* ZEBRA_CACHED_STATE_DIR: /
* LIGHTWALLETD_DATA_DIR: /

This seems like it could be confusing and error prone.
(But I think `zebrad` and `lightwalletd` set up their own subdirectories anyway, so it should be fine, as long as they don't try to delete their own state directories.)

Can you document what is happening, and why it's implemented that way?

Reply:
Explained with [docs(ci): explain disk mounting logic](https://github.com/ZcashFoundation/zebra/pull/4268/commits/bdb217b3b8b760e834c537fe81ad39dc40bf4f2c)